lib/definitions/errors.js

```diff
@@ -139,6 +139,17 @@ By default the \`repositoryUrl\` option is retrieved from the \`repository\` pro
   };
 }
 
+export function EMISMATCHGITHUBURL() {
+  return {
+    message: "The git repository URL mismatches the GitHub URL.",
```

Comment:
@gr2m @babblebey it would be a lot more convenient to include the two values that didn't match in an error message like this